Translations/Portuguese/Basic Information.md

```diff
@@ -1,2 +1,20 @@
 This is the link  to the slack bot wiki basic information page to be translated.
-[Basic Information](https://github.com/chaoss/chaoss-slack-bot/wiki/Basic-Information)
\ No newline at end of file
+[Basic Information](https://github.com/chaoss/chaoss-slack-bot/wiki/Basic-Information)
+
+> ### Informação básica
```

Comment:
```suggestion
> ### Informações básica
```

Reply:
I have already made the recommended changes